lago/export.py

```diff
@@ -185,9 +189,18 @@ def rebase(self):
                 # this identifier will be used later by Lago in order
                 # to resolve and download the base image
                 parent = self.src_qemu_info[0]['backing-filename']
-                parent = './{}'.format(
-                    os.path.basename(parent.split(':', 1)[1])
-                )
+
+                # Hack for working with lago images naming convention
+                # For example: /var/lib/lago/store/phx_repo:el7.3-base:v1
+                # Extract only the image name and the version
+                # (in the example el7.3-base:v1)
+                parent = os.path.basename(parent)
+                try:
+                    parent = parent.split(':', 1)[1]
```

Comment:
OK - I guess both are viable. personally I  prefer using regex named groups that give you a clue what you extracted from the string.